packages/api/src/tables/stores-factory.ts

```diff
@@ -0,0 +1,23 @@
+import { Datastore } from "@google-cloud/datastore";
+import { tables, stores } from "@uma/sdk";
+import { empStats, empStatsHistory, lsps } from ".";
+
+const { GoogleDatastore } = stores;
+const datastoreClient = new Datastore();
+
+export const datastores = {
+  empsActiveStore: GoogleDatastore<string, tables.emps.Data>("Active Emp", datastoreClient),
```

Comment:
this file in general looks great, personally though i would not include "Stores" in the name of each propery, since you know that context from the import.

I would also suggest potentially doing this init in a function that takes in a datastoreClient parameter, so that could be configured independently by the caller